content/sql/concepts/window-functions/terms/lag/lag.md

```diff
@@ -29,33 +32,106 @@ OVER ( [ partition_by ] order_by )
 - `partition_by` - Allows the result set to be grouped based on a column. This is an optional argument, if not specified the result set will be treated as a single group.
 - `order_by` - Determines the order of the result set. If `partition_by` is specified, it will order the grouped data instead.
 
-## Example
+## Examples
 
-The example below uses the `LAG()` function to create the new column `previous_age` based on values in the previous row of the `Users` table.
+The examples below use the table Sales, which contains orders of different products that are sold on a particular date and their price that they are sold to.
```

Comment:
We should use the table name inside backticks to signify the exact name used in the code as well. Also, we should use the word table after the table name for better readability.

Also, some minor adjustments can make it much easier to follow like this -

```suggestion
The examples below use the Sales table, which contains orders for different products sold on specific dates, along with their prices.
```